### PR TITLE
Added the debug toolbar combos to the ignore list

### LINKED ISSUE #4: Shows Debug.LocationToolbar.StackFrameCombo Repeatedly
This is the same as #3 except it is caused by the dropdowns in the Debug Location toolbar when debugging and stopped at a breakpoint.

```
2016-05-24 3:56:28 PM: Debug.LocationToolbar.StackFrameCombo (Ctrl+7)
2016-05-24 3:56:28 PM: Debug.LocationToolbar.ProcessCombo (Ctrl+5)
2016-05-24 3:56:28 PM: Debug.LocationToolbar.ThreadCombo (Ctrl+6)
2016-05-24 3:56:28 PM: Debug.LocationToolbar.StackFrameCombo (Ctrl+7)
2016-05-24 3:56:29 PM: Debug.LocationToolbar.ProcessCombo (Ctrl+5)
2016-05-24 3:56:29 PM: Debug.LocationToolbar.ThreadCombo (Ctrl+6)
2016-05-24 3:56:29 PM: Debug.LocationToolbar.StackFrameCombo (Ctrl+7)
2016-05-24 3:56:29 PM: Debug.LocationToolbar.ProcessCombo (Ctrl+5)
2016-05-24 3:56:29 PM: Debug.LocationToolbar.ThreadCombo (Ctrl+6)
```

I am preparing a pull request. It should be ready soon. I am just adding to your ignore list, but if more of these are found, you may want to filter on commands ending in Combo.

### PR DESCRIPTION
**Pull request by rprouse**
Fixes #4

### CODE REVIEW

File: src/ShowTheShortcut.csproj [hunk 1]
```diff
@@ -27,6 +27,9 @@
     <RootNamespace>ShowTheShortcut</RootNamespace>
     <AssemblyName>ShowTheShortcut</AssemblyName>
     <TargetFrameworkVersion>v4.5.1</TargetFrameworkVersion>
+    <StartAction>Program</StartAction>
+    <StartProgram>$(DevEnvDir)\devenv.exe</StartProgram>
+    <StartArguments>/rootsuffix Exp</StartArguments>
```
Comment (rprouse) [inline src/ShowTheShortcut.csproj]: Unrelated change to make it unnecessary to manually setup debugging every time you clone. I added as I was testing.

File: src/source.extension.vsixmanifest [hunk 2]
```diff
@@ -4,7 +4,7 @@
         <Identity Id="9da28329-f9d5-4f18-91c3-d3285b103d1a" Version="1.4" Language="en-US" Publisher="Mads Kristensen" />
         <DisplayName>Learn the Shortcut</DisplayName>
         <Description xml:space="preserve">Shows how easy you can make the same action using only the keyboard. Displays the keyboard shortcut for any command that you execute to help you learn the shortcuts you need the most.</Description>
-        <MoreInfo>https://github.com/madskristensen/ShowTheShortut</MoreInfo>
+        <MoreInfo>https://github.com/madskristensen/ShowTheShortcut</MoreInfo>
```
Comment (rprouse) [inline src/source.extension.vsixmanifest]: Noticed the 404 when I was walking out to this repo so I fixed it.

**Comment by madskristensen** [comment]
Awesome. Thanks for this!!! Much appreciated
